src/logic/StatePair.java

```diff
@@ -32,12 +34,14 @@ public void setNode(GraphNode node) {
     }
 
     public String prettyPrint() {
```

Comment:
Could you add some documentation? Three points that might be useful to know later:
1. It should make clear what the difference is between `prettyPrint()` and `toString()`.
2. It should mention that `left.getInvariant()` equals `right.getInvariant()`, so only one single guard is printed.
3. It should mention that the zone is printed as a guard (not as a CDD or DBM)

Reply:
I have added some comments now. I hope they suffice